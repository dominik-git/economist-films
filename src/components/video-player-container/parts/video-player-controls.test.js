// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayerControls from './video-player-controls';

test('VideoPlayerControls renders correctly when isVideoPlaying is true', () => {
  const tree : string =
  renderer.create(
    <VideoPlayerControls
      playVideo={() => {}}
      pauseVideo={() => {}}
      fastForward={() => {}}
      fastRewind={() => {}}
      isVideoPlaying
      progress={0}
    />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('VideoPlayerControls renders correctly when isVideoPlaying is false', () => {
  const tree : string =
  renderer.create(
    <VideoPlayerControls
      playVideo={() => {}}
      pauseVideo={() => {}}
      fastForward={() => {}}
      fastRewind={() => {}}
      isVideoPlaying={false}
      progress={0}
    />).toJSON();
  expect(tree).toMatchSnapshot();
});
