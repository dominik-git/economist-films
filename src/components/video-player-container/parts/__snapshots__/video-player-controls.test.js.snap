// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerControls renders correctly when isVideoPlaying is false 1`] = `
<div
  className="player-interface"
>
  <div
    className="player-interface__progress-bar-container"
  >
    <div
      className="player-interface__progress-bar"
      style={
        Object {
          "width": "0%",
        }
      }
    />
  </div>
  <div
    className="player-interface__navigation-wrapper"
  >
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="right"
        className="player-interface-navigation-icons"
        src="RW.svg"
      />
    </button>
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="Play/Pause"
        className="player-interface-navigation-icons"
        src="Play.svg"
      />
    </button>
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="left"
        className="player-interface-navigation-icons"
        src="FF.svg"
      />
    </button>
  </div>
</div>
`;

exports[`VideoPlayerControls renders correctly when isVideoPlaying is true 1`] = `
<div
  className="player-interface"
>
  <div
    className="player-interface__progress-bar-container"
  >
    <div
      className="player-interface__progress-bar"
      style={
        Object {
          "width": "0%",
        }
      }
    />
  </div>
  <div
    className="player-interface__navigation-wrapper"
  >
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="right"
        className="player-interface-navigation-icons"
        src="RW.svg"
      />
    </button>
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="Play/Pause"
        className="player-interface-navigation-icons"
        src="Pause.svg"
      />
    </button>
    <button
      className="player-interface__navigation-control"
      onClick={[Function]}
    >
      <img
        alt="left"
        className="player-interface-navigation-icons"
        src="FF.svg"
      />
    </button>
  </div>
</div>
`;
